_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __ } from 'i18n-calypso';
+
+class ProductExpiration extends React.PureComponent {
+	static propTypes = {
+		expiryDate: PropTypes.string.isRequired,
+		purchaseDate: PropTypes.string,
+		isRefundable: PropTypes.bool,
+		dateFormat: PropTypes.string,
+		fallback: PropTypes.string,
```

Comment:
I think we can totally remove the `fallback` prop and let components that render component handle this as they like.

Reply:
Fixed in 84727755f71988299f54bf4e06f3dcc99d6171ab